bip-0003.md

```diff
@@ -645,13 +646,13 @@ feedback, and helpful comments.
     ready to recommend their BIP for adoption. The term "ready" was also considered, but considered too subjective.
 [^rejection]: **Why can proposals remain in Draft or Complete indefinitely?**  
     The automatic 3-year timeout of BIPs has led to some disagreement in the past and seems unnecessary in cases where
-    the authors are still active in the community and still consider their idea worth pursuing. On the other hand,
-    Draft proposals that appear stale may be tested and cleared out after only one year which should achieve the main goal of
+    the authors are remain active in the community and still consider their idea worth pursuing. On the other hand,
```

Comment:
```suggestion
    the authors remain active in the community and still consider their idea worth pursuing. On the other hand,
```

Reply:
Ah yes, of course. Thanks for catching that.